source/vstd/std_specs/vecdeque.rs

```diff
@@ -0,0 +1,374 @@
+/// This code adds specifications for the standard-library type
+/// `std::collections::VecDeque`.
+use super::super::prelude::*;
+
+use alloc::collections::vec_deque::Iter;
+use alloc::collections::vec_deque::VecDeque;
+use core::alloc::Allocator;
+use core::clone::Clone;
+use core::ops::Index;
+use core::option::Option;
+use core::option::Option::None;
+
+verus! {
+
+#[verifier::external_type_specification]
+#[verifier::external_body]
+#[verifier::accept_recursive_types(T)]
+#[verifier::reject_recursive_types(A)]
+pub struct ExVecDeque<T, A: Allocator>(VecDeque<T, A>);
+
+// this is a bit of a hack; verus treats Global specially already,
+// but putting this here helps Verus pick up all the trait impls for Global
+#[verifier::external_type_specification]
+#[verifier::external_body]
+pub struct ExGlobal(alloc::alloc::Global);
```

Comment:
I just moved `pub struct ExGlobal(alloc::alloc::Global);` to a different file, so it's no longer necessary to include it in `vec.rs` (or `vecdeque.rs`).